src/versionSocketConnection.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { EventEmitter } from 'events';
+import WebSocket from 'ws';
+import { WebSocketEvent } from './common/webviewEvents';
+
+export type IDevToolsPostMessageCallback = (e: WebSocketEvent, message?: string) => void;
+
+export interface BrowserVersionCdpResponse {
+   id: number;
+   result?: {
+       product?: string;
+       revision?: string;
+   }
+}
+
+const MIN_SUPPORTED_VERSION = '94.0.975.0';
```

Comment:
TODO: update to actual min version

Reply:
Going to leave as is for now to facilitate testing, but this should be updated to the first canary after the last polyfill migrations land